global/settings/types/preferencekeys.h

```diff
@@ -52,6 +52,7 @@
 #define PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED         "application/startup/telemetry_access_requested"
 #define PREF_APP_TELEMETRY_ALLOWED                          "application/telemetry/allowed"
 #define PREF_APP_BACKUP_GENERATE_BACKUP                     "application/backup/generateBackup"
+#define PREF_APP_BACKUP_PATH                                "application/backup/path"
```

Comment:
This is a subfolder, not a path, so I suggest you change this to:

`#define PREF_APP_BACKUP_SUBFOLDER                           "application/backup/subfolder"`